common/core/desktop/src/kmx/kmx_file.cpp

```diff
@@ -59,7 +59,7 @@ const unsigned long CRCTable[256] = {
  * and returns the updated value.
  */
 
-unsigned long CalculateBufferCRC(unsigned long count, KMX_BYTE *p)
+static unsigned long CalculateBufferCRC(unsigned long count, KMX_BYTE *p)
```

Comment:
Curious why you made this static? Not that that's a real problem as such, but just curious!

Reply:
Once we fixed the common core to be actual static library, I then got a build error as this had a namespace clash with the function of the same name in Compiler.cpp.   Looking at the this kmx_file.cpp use of it was just in this module so I it was easiest to just make it officially private.  I also knew we had talked about there being some consolidation of complier.h (Mentioned when we split "keyman64.h" and keymanengine.h"). So the longer term consolidation was coming.  I can do that if you prefer but would rather do it in another PR.

Reply:
Doing that in another PR is good.